.gitignore

```diff
@@ -0,0 +1,3 @@
+package-lock.json
+package.json
```

Comment:
These files should be in the root of your project - it is only node_modules that should be ignored at present

Reply:
Incidentally - this is why you had a few struggles in getting you project to get off the ground.